nixos/modules/installer/cd-dvd/iso-image.nix

```
@@ -512,9 +512,10 @@ in
                 + lib.optionalString isAarch "-Xbcj arm"
                 + lib.optionalString (isPower && is32bit && isBigEndian) "-Xbcj powerpc"
                 + lib.optionalString (isSparc) "-Xbcj sparc";
-      type = lib.types.str;
+      type = lib.types.nullOr lib.types.str;
       description = lib.mdDoc ''
         Compression settings to use for the squashfs nix store.
+        `null` disables compression.
       '';
       example = "zstd -Xcompression-level 6";
```

```suggestion
      example = "-comp zstd -Xcompression-level 6";
```
would it make things easier if we just removed the default -compo and allowed to set this to empty string?

---

I feel like if we were to do that we might as well do the other suggestion of having a list of "mksquashfs extra args". And I'm still nervous about other external users that are passing this flag into `nixos/lib/make-squashfs.nix`.